src/Conticket/ApiBundle/Controller/EventController.php

```diff
@@ -62,49 +69,52 @@ public function cgetAction($limit, $offset)
      */
     public function getAction($id)
     {
-        return ['event' => $this->getOr404($id)];
```

Comment:
Why you removed 'event' here and keep 'order' in https://github.com/PHPeste/conticket/pull/26/files#diff-93060620b8575bcef04f8c5ccf5aae58R63?